assignments/callbacks.js

```diff
@@ -24,19 +25,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr);
```

Comment:
The callback here is supposed to return the length of the array. 

`return cb(arr.length);